myadsp/tests/test_utils.py

```diff
@@ -579,30 +579,33 @@ def test_get_first_author_formatted(self):
     def test_payload_to_plain(self):
 
         formatted_payload = utils.payload_to_plain(payload)
-
+    
         split_payload = formatted_payload.split('\n')
         self.assertEqual(split_payload[0].strip(), 'Query 1 (https://ui.adsabs.harvard.edu/search/q=bibstem%3Aarxiv?utm_source=myads&utm_medium=email&utm_campaign=type:general&utm_term=123&utm_content=queryurl)')
         self.assertEqual(split_payload[1].strip(), '"VizieR Online Data Catalog: Spectroscopy of M81 globular ' +
                                                     'clusters," Nantais, J and Huchra, J (2012yCat..51392620N)')
 
     def test_payload_to_html(self):
-
+        
         formatted_payload = utils.payload_to_html(payload, col=1, email_address="test@tester.com")
 
         split_payload = formatted_payload.split('\n')
-        self.assertIn(u'templateColumnContainer"', split_payload[77])
-        self.assertEqual(split_payload[79].strip(),
+        self.assertIn(u'templateColumnContainer"', split_payload[86])
+
+        self.assertEqual(split_payload[91].strip(),
                           u'<h3><a href="https://ui.adsabs.harvard.edu/search/q=bibstem%3Aarxiv?utm_source=myads&amp;utm_medium=email&amp;utm_campaign=type:general&amp;utm_term=123&amp;utm_content=queryurl" title="" style="text-decoration: none; color: #000000; font-weight: bold;">Query 1</a></h3>')
-        self.assertIn(u'href="https://ui.adsabs.harvard.edu/abs/2012yCat..51392620N/abstract?utm_source=myads&amp;utm_medium=email&amp;utm_campaign=type:general&amp;utm_term=123&amp;utm_content=rank:1"', split_payload[98])
+        
+        # self.assertIn(u'href="https://ui.adsabs.harvard.edu/abs/2012yCat..51392620N/abstract?utm_source=myads&amp;utm_medium=email&amp;utm_campaign=type:general&amp;utm_term=123&amp;utm_content=rank:1"', split_payload[110])
```

Comment:
Do we still want to hold onto this commented line or is it good to delete?

Reply:
I made a small change to it and uncommented it.